examples/todo-knex/index.js

```diff
@@ -0,0 +1,27 @@
+const { Keystone } = require('@keystonejs/keystone');
+const { KnexAdapter } = require('@keystonejs/adapter-knex');
+const { Text } = require('@keystonejs/fields');
+const { GraphQLApp } = require('@keystonejs/app-graphql');
+const { AdminUIApp } = require('@keystonejs/app-admin-ui');
+const { StaticApp } = require('@keystonejs/app-static');
+
+const keystone = new Keystone({
+  adapter: new KnexAdapter({ knexOptions: { connection: 'postgres://localhost/to-do' } }),
```

Comment:
We should give some instructions on how to create this database in the README.md